### ISSUE TITLE
Versioning is not fuctioning

### ISSUE THREAD

**Issue by ostasevych** [ONLYOFFICE/CommunityServer#360]
Hi! I have NC 23.0.0.2 with installed Community version of Onlyoffice. When clicking on Collaboration->Version History menu I see that the list of versions is shown, and the notice Opening Document appears, which is not finishing anywhen. The app stucks until refresh of the page.

There's nothing in the logs of the Nextcloud.

![image](https://user-images.githubusercontent.com/5199030/154916213-f2692302-a30b-4d92-bccf-ffb46d58b7d1.png)

**Comment by ShockwaveNN** [comment]
Hi, to clarify

> Community version of Onlyoffice

Are you using Nextcloud's Community DocumentServer?

Please note that Community Document Server was developed by Nextcloud Team, if you have any issues with it, please contact them in the appropriate [GitHub repository](https://github.com/nextcloud/documentserver_community). 

If you are using our official method of integration (Nextcloud + dedicated Document Server instance as per our installation [guide](https://api.onlyoffice.com/editors/nextcloud)), we will be glad to answer your questions. Please see more info on our [forum](https://forum.onlyoffice.com/t/community-document-server-app-in-nextcloud-18/268).

**Comment by ostasevych** [comment]
> Hi, to clarify
> 
> > Community version of Onlyoffice
> 
> Are you using Nextcloud's Community DocumentServer?
> 
> Please note that Community Document Server was developed by Nextcloud Team, if you have any issues with it, please contact them in the appropriate [GitHub repository](https://github.com/nextcloud/documentserver_community).
> 
> If you are using our official method of integration (Nextcloud + dedicated Document Server instance as per our installation [guide](https://api.onlyoffice.com/editors/nextcloud)), we will be glad to answer your questions. Please see more info on our [forum](https://forum.onlyoffice.com/t/community-document-server-app-in-nextcloud-18/268).

Yes, this is NC installation. Thanks for additional clarifications.

**Comment by ShockwaveNN** [comment]
In that case I'll close this issue
